tox.ini

```diff
@@ -25,6 +26,7 @@ deps =
     django42: django~=4.2.0
     django50: django~=5.0.0
     django51: django~=5.1.0
+    django51: django~=5.2.0
```

Comment:
Wait shouldn't this be 52?

Reply:
Yes, you are absolutely right, I was starting to look at `django-zendesk-tickets` and just spotted it! I'll change.

Reply:
https://github.com/ministryofjustice/django-moj-irat/pull/16

(I'm surprised tox didn't complain/fail)